README.md

```diff
@@ -96,11 +104,6 @@ The following features are not implemented yet:
 
 There a lot more work to be done on the UI.
 
-## Known Issues
-
-Terminal colors are not reset correctly on exit. This should have a smaller
```

Comment:
FYI this is not fixed, apparently there's no way to automatically reset colors. I was thinking about manually saving/restoring them, but probably not critical at this point so we can clean this up.

Reply:
oh! I thought I saw a fix added, but I guess it was just not changing the low colors